c2corg_api/tests/views/test_articles.py

```diff
@@ -0,0 +1,332 @@
+from c2corg_api.models.article import ArchiveArticle, Article, ARTICLE_TYPE
+from c2corg_api.tests.search import reset_search_index
+
+from c2corg_api.models.document import (
+    ArchiveDocumentLocale, DocumentLocale)
+from c2corg_api.views.document import DocumentRest
+
+from c2corg_api.tests.views import BaseDocumentTestRest
+from c2corg_common.attributes import quality_types
+
+
+class TestArticleRest(BaseDocumentTestRest):
+
+    def setUp(self):  # noqa
+        self.set_prefix_and_model(
+            "/articles", ARTICLE_TYPE, Article, ArchiveArticle,
+            ArchiveDocumentLocale)
+        BaseDocumentTestRest.setUp(self)
+        self._add_test_data()
+
+    def test_get_collection(self):
+        body = self.get_collection()
+        doc = body['documents'][0]
+        self.assertNotIn('geometry', doc)
+
+    def test_get_collection_paginated(self):
+        self.app.get("/articles?offset=invalid", status=400)
+
+        self.assertResultsEqual(
+            self.get_collection({'offset': 0, 'limit': 0}), [], 4)
+
+        self.assertResultsEqual(
+            self.get_collection({'offset': 0, 'limit': 1}),
+            [self.article4.document_id], 4)
+        self.assertResultsEqual(
+            self.get_collection({'offset': 0, 'limit': 2}),
+            [self.article4.document_id, self.article3.document_id], 4)
+        self.assertResultsEqual(
+            self.get_collection({'offset': 1, 'limit': 2}),
+            [self.article3.document_id, self.article2.document_id], 4)
+
+    def test_get_collection_lang(self):
+        self.get_collection_lang()
+
+    def test_get_collection_search(self):
+        reset_search_index(self.session)
+
+        self.assertResultsEqual(
+            self.get_collection_search({'l': 'en'}),
+            [self.article4.document_id, self.article1.document_id], 2)
+
+        self.assertResultsEqual(
+            self.get_collection_search({'act': ['hiking']}),
+            [self.article4.document_id, self.article3.document_id,
+             self.article2.document_id, self.article1.document_id], 4)
+
+    def test_get(self):
+        body = self.get(self.article1)
+        self.assertNotIn('article', body)
+
+    def test_get_lang(self):
+        self.get_lang(self.article1)
+
+    def test_get_new_lang(self):
+        self.get_new_lang(self.article1)
+
+    def test_get_404(self):
+        self.get_404()
+
+    def test_post_error(self):
+        body = self.post_error({}, user='moderator')
+        errors = body.get('errors')
+        print(errors)
```

Comment:
I did "git rebase -i HEAD~4", put edit to that commit instead of pick, deleted print() in my code, git added that file, then "git commit --amend" and "git rebase --continue" and pushed. it should be ok now.